textalyze.js

```diff
@@ -14,10 +6,27 @@
 function itemCounts(array) {
   let counts = new Map();
 
-  // Your code here.
-  // Run 'npm test' to see what tests need to pass.
+  array.forEach(element => {
+    
+    if (!counts.has(element))
+      counts.set(element, 1);
+    else 
+      counts.set(element, counts.get(element) + 1);
+  });
 
   return counts;
 }
 
 module.exports = { itemCounts };
+
+//
+// Running...
+//
+
+var array = ['a', 'a', 'a', 'b', 'b', 'c'];
```

Comment:
We'd like this file to be both included as a module and run via `npm start`.  Check out https://stackoverflow.com/questions/4981891/node-js-equivalent-of-pythons-if-name-main for how to do this.

You want

```javascript
if (require.main == module) {
  const array = ['a', 'a', 'a', ...];
}
```

Reply:
Use `let` or `const`.  I've set up the project with an ESLint file — set up you editor to use eslint or run it manually with `npx eslint`.